styles/style.css

```diff
@@ -0,0 +1,37 @@
+header {
+    background-color:rgb(31, 113, 101);
+}
+body{
+    font-family:Georgia, 'Times New Roman', Times, serif
+}
+#main {
+    background-image: url('https://thumbs.gfycat.com/QuestionablePeriodicGharial-size_restricted.gif');
+    background-size: cover;
+    height: 80vh;
+    padding: 0;
+    margin: 0;
+}
+h1 {
+    margin-top: 1.2 em;
+    text-align: center;
+}
+
+.top-bottom-links{
+    padding: 1em;
+    display: flex;
+    flex: wrap;
+    flex-direction: row;
+    justify-content: space-evenly;
+    list-style-type: none;
+    background-color:rgb(114, 106, 233);
+    
+}
+#about-picture {
+    display: inline-block;
+    width: 22%;
+}
+.about-me{
+    float: left;
```

Comment:
Try to avoid using float if possible, but where you _do_ use it, it's generally intended to reposition something (like an image) with some main text flowing normally, and the item being floated being moved to one side. This is often used with images presented in the flow of some longer text, as the text will flow around the floated image. So your use here is really close, but rather than floating the text left, I would float your image right.